### ISSUE TITLE
"Offset" is ignored when 'smooth' is true.

### ISSUE THREAD

**Issue by MitchEff** [locomotivemtl/locomotive-scroll#231]
We've just written our own in-view detection scripts to get around this on all our sites, but it may be useful to know that no value whatsoever is respected for `offset` (in initialisation) or `data-scroll-offset` (on elements themselves) when `smooth` is true. Numbers, percentages etc are ignored and it just adds its own `inview` class whenever it feels like it.

Has anyone else experienced that?

**Comment by tigerchick** [comment]
Have you tried specifying 2 values eg: 
data-scroll-offset="0, 20%"

**Comment by MitchEff** [comment]
@tigerchick yeah, works perfectly without 'smooth' on, but otherwise any variations in how we set offset values is ignored.
This is both in the initialisation props and data attributes - seems to make no difference. 

We're using `getBoundingClientRect` to run our own in-view detection in the interim, but it'd be stellar if we could remove it as it's not overly performant.
